recipes-bsp/u-boot/u-boot-iot2050-2021.x-upstream.inc

```diff
@@ -54,7 +80,13 @@ DEBIAN_BUILD_DEPENDS =. "trusted-firmware-a-iot2050, optee-os-iot2050, \
 
 do_prepare_build_append() {
     cd ${S}
-    ln -sf ../prebuild/* ${S}
+    if [ ${SOC_VERSION} = "sr1" ]; then
+        ln -sf ../prebuild/prufw_sr1/* ${S}
+        ln -sf ../prebuild/sysfw_sr1/* ${S}
```

Comment:
Just curious, we do have another am65x-sr2-txpru0-prueth-fw.elf and am65x-sr2-txpru0-prueth-fw.elf for kernel, but not for u-boot. Doesn't u-boot need it?

Reply:
I don't know. I don't have tested the eth function under u-boot.
The problem needs to be investigated

Reply:
Sure, anyway atm eth is not on the critical path to sr2 development, we could address that later.